frontend/src/app/main/ui/auth/login.cljs

```diff
@@ -138,9 +138,9 @@
                            :on-success on-success})]
              (st/emit! (du/login-with-ldap params)))))
 
-        on-recovery-request
-        (mf/use-fn
-         #(st/emit! (rt/nav :auth-recovery-request)))]
+        on-recovery-request (or on-recovery-request
+                                (mf/use-fn
```

Comment:
You can't put react-hooks on conditionals, they should be defined statically.

Reply:
fixed